### PR TITLE
Test adding new training layers for cell curation

### PR DESCRIPTION
**Pull request by dstansby**
This checks that adding new training layers:
- Adds two layers
- That the layers are both Points layers
- That the layers are named correctly

### CODE REVIEW

File: cellfinder_napari/tests/test_curation.py [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
+def test_add_new_training_layers(curation_widget):
+    viewer = curation_widget.viewer
+    layers = viewer.layers
+    # Check that layers list starts off empty
+    assert len(layers) == 0
+    curation_widget.add_training_data()
+    assert len(layers) == 2
+
+    for layer in layers:
+        assert isinstance(layer, Points)
```
Comment (paddyroddy) [inline cellfinder_napari/tests/test_curation.py:29]:
```suggestion
    assert all(isinstance(layer, Points) for layer in layers)
```

**Review by paddyroddy — APPROVED** [review]
👍 , one suggestion

File: cellfinder_napari/tests/test_curation.py (hunk 1 above)
Comment (dstansby) [reply cellfinder_napari/tests/test_curation.py:29]: Ah that's much nicer 👍

Comment (paddyroddy) [reply cellfinder_napari/tests/test_curation.py]: Probably faster too (granted it is only two entries 😁)

**Pull request merged**